planet/clients/subscriptions.py

```diff
@@ -310,8 +310,7 @@ async def get_results_csv(
                                           "queued",
                                           "processing",
                                           "failed",
-                                          "success"]]] = None,
-        limit: int = 100,
```

Comment:
We had a Slack discussion and found that this arg is unused, undocumented and was never required. This theoretically could be a breaking change for somebody, but removing it seems reasonable